client/src/components/form/ReduxFormUtils.jsx

```diff
@@ -204,3 +205,26 @@ export const validateURL = value => {
 
 export const hasError = errors =>
     _keys(errors).find(key => _size(errors[key]) > 0) !== undefined
+
+const SemanticVersionError = () => {
+    return (
+        <div style={{ whiteSpace: 'pre-wrap' }}>
+            Must be a semantic version. Use the form <span>x.x.x.</span>
+            <br />
+            <a
+                href="https://docs.npmjs.com/about-semantic-versioning"
+                target="_blank"
+            >
+                Click here
+            </a>{' '}
+            for more information
```

Comment:
Side note - seems like we don't support translations in the App Hub?  We probably want to eventually

Reply:
That's correct. Something that's been discussed for a long time, but I didn't find an issue for it - will create one! It shouldn't be too bad to add, but I don't know if we want to use the same workflow as DHIS2-apps here?

Reply:
Issues for that here: https://jira.dhis2.org/projects/HUB/issues/HUB-93